packages/11ty/_includes/components/citation/page.js

```diff
@@ -43,7 +42,7 @@ module.exports = function (eleventyConfig) {
       'publisher-place': publishers[0].location,
       title: pageTitle(page.data),
       type: 'chapter',
-      URL: new URL(page.url, baseURL)
+      URL: new URL(page.url, url)
```

Comment:
Should this be the `URL` object or its string representation?
```suggestion
      url: new URL(page.url, url).toString()
```